New lesson: analisis-redes-sociales-teatro-2

jenniferisasi opened the pull request:
Publish new ES lección ✨ analisis-redes-sociales-teatro-2

Closes #3096 

### Checklist

- [x] Assign yourself in the "Assignees" menu
- [x] Add the appropriate "Label"
- [x] If this PR closes an Issue, add the phrase `Closes #ISSUENUMBER` to your summary above
- [x] Ensure the status checks pass: if you have difficulty fixing build errors, please contact our Publishing Assistant @anisa-hawes 
- [x] Check the Netlify Preview: navigate to netlify/ph-preview/deploy-preview and click 'details' (at right)
- [x] Assign at least one individual or team to "Reviewers"
  - [ ] if the text needs to be translated, please follow the [translation request guidelines](https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines), then assign the relevant language team(s) as "Reviewers" and tag both the team as well as the managing editor in your PR.

jenniferisasi committed:
adding lesson analisis-redes-sociales-teatro-2

jenniferisasi committed:
add images for analisis-redes-sociales-teatro-2

jenniferisasi committed:
add avatar for analisis-redes-sociales-teatro-2

jenniferisasi committed:
add original image for analisis-redes-sociales-teatro-2

anisa-hawes commented:
Hello @jenniferisasi,

The build checks for Part-2 will fail until Part-1 is merged because it will require David's bio in `ph_authors.yml`. 

Our other option is to prepare both in one branch. What do you think?

anisa-hawes committed:
Update analisis-redes-sociales-teatro-2.md

- Adjust `date:`
- Adjust line 37 to clarify (thank you @jenniferisasi)

anisa-hawes committed:
Move analisis-redes-sociales-teatro-2.md

Move inside /es/lecciones directory.

Commit:
Merge branch 'gh-pages' into Issue-3096

anisa-hawes committed:
Update analisis-redes-sociales-teatro-2.md

- Lowercase 'parte' in title (for consistence with parte 1)
- Remove duplicate link at line 27

anisa-hawes committed:
Update analisis-redes-sociales-teatro-2.md

- Shorten `abstract:` (thank you @jenniferisasi).

jenniferisasi commented:
Preview looks 🔥
